src/main/java/com/gtnewhorizons/angelica/glsm/stacks/BooleanStateStack.java

```diff
@@ -32,7 +32,6 @@ public BooleanStateStack pop() {
         }
 
         set(stack[--pointer]);
-        this.setEnabled(this.enabled);
```

Comment:
Tests on AMD fail without this -- we'll need to make this more targeted to 1) AMD 2) GL_BLEND

Reply:
But on a related note - thanks for tracking down the bug and the fix